### ISSUE TITLE
Build is throwing error - Step 3/3 : COPY crypto /orderer/crypto COPY failed: stat /var/lib/docker/tmp/docker-builder825722007/crypto: no such file or directory

### ISSUE THREAD

**Issue by biswroy4** [IBM/build-blockchain-insurance-app#72]
In the Build phase, following error is getting displayed:
Sending build context to Docker daemon  2.048kB
Step 1/3 : FROM docker.io/hyperledger/fabric-orderer:latest
 ---> ec4ca236d3d4
Step 2/3 : RUN mkdir /orderer
 ---> Running in 6eddd5d61ee8
Removing intermediate container 6eddd5d61ee8
 ---> 3030cf93b3b2
Step 3/3 : COPY crypto /orderer/crypto
COPY failed: stat /var/lib/docker/tmp/docker-builder825722007/crypto: no such file or directory
Creating network "build-blockchain-insurance-app_default" with the default driver
Pulling insurance-ca (insurance-ca:)...
The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

**Comment by sblisztomaniac** [comment]
> In the Build phase, following error is getting displayed:
> Sending build context to Docker daemon 2.048kB
> Step 1/3 : FROM docker.io/hyperledger/fabric-orderer:latest
> ---> ec4ca236d3d4
> Step 2/3 : RUN mkdir /orderer
> ---> Running in 6eddd5d61ee8
> Removing intermediate container 6eddd5d61ee8
> ---> 3030cf93b3b2
> Step 3/3 : COPY crypto /orderer/crypto
> COPY failed: stat /var/lib/docker/tmp/docker-builder825722007/crypto: no such file or directory
> Creating network "build-blockchain-insurance-app_default" with the default driver
> Pulling insurance-ca (insurance-ca:)...
> The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

refer issue #68

**Comment by biswroy4** [comment]
Still getting the same message:
Sending build context to Docker daemon  2.048kB
Step 1/3 : FROM docker.io/hyperledger/fabric-orderer:latest
 ---> ec4ca236d3d4
Step 2/3 : RUN mkdir /orderer
 ---> Using cache
 ---> 3030cf93b3b2
Step 3/3 : COPY crypto /orderer/crypto
COPY failed: stat /var/lib/docker/tmp/docker-builder246830254/crypto: no such file or directory
Pulling insurance-ca (insurance-ca:)...
The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]

My Web/Dockerfile looks like: FROM ubuntu:trusty
RUN apt update && apt install -y build-essential
FROM docker.io/library/node:8.11.3
ENV NODE_ENV production
ENV PORT 3000
ENV DOCKER_SOCKET_PATH /host/var/run/docker.sock
ENV DOCKER_CCENV_IMAGE hyperledger/fabric-ccenv:latest
RUN mkdir /app
COPY . /app
WORKDIR /app
RUN npm i && npm i --only=dev /
      && npm run build /
      && npm prune /
      && apt remove -y build-essential
EXPOSE 3000
CMD ["npm", "run", "serve"]

My Peer-base.yaml file:
version: '2'

services:
  peer-base:
    environment:
    - CORE_VM_ENDPOINT=unix:///host/var/run/docker.sock
    - CORE_VM_DOCKER_HOSTCONFIG_NETWORKMODE=buildblockchaininsuranceapp_default
    - CORE_LOGGING_LEVEL=DEBUG
    - CORE_PEER_TLS_ENABLED=true
    - CORE_PEER_ENDORSER_ENABLED=true
    - CORE_PEER_GOSSIP_USELEADERELECTION=true
    - CORE_PEER_GOSSIP_ORGLEADER=false
    - CORE_PEER_PROFILE_ENABLED=true
    - CORE_PEER_MSPCONFIGPATH=/peer/crypto/msp
    - CORE_PEER_TLS_CERT_FILE=/peer/crypto/tls/server.crt
    - CORE_PEER_TLS_KEY_FILE=/peer/crypto/tls/server.key
    - CORE_PEER_TLS_ROOTCERT_FILE=/peer/crypto/tls/ca.crt
    working_dir: /peer
    command: peer node start
    volumes:
    - /var/run/:/host/var/run/

**Comment by sblisztomaniac** [comment]
what does docker network ls give you? 
check  whether the  newtork name and  - CORE_VM_DOCKER_HOSTCONFIG_NETWORKMODE=buildblockchaininsuranceapp_default 
are the same.

**Comment by biswroy4** [comment]
It gives me following:

C:\Users\BiswajitRoy\build-blockchain-insurance-app>docker network ls
NETWORK ID          NAME                                     DRIVER              SCOPE
8a53bcfdaaba        bridge                                   bridge              local
e04c7c6e7850        build-blockchain-insurance-app_default   bridge              local
639da61ae87a        host                                     host                local
c02ffcb8c4a0        none                                     null                local

**Comment by biswroy4** [comment]
I removed the network name from peer-base.yaml file. Still it is throwing the error:
Sending build context to Docker daemon  2.048kB
Step 1/3 : FROM docker.io/hyperledger/fabric-orderer:latest
 ---> ec4ca236d3d4
Step 2/3 : RUN mkdir /orderer
 ---> Using cache
 ---> 3030cf93b3b2
Step 3/3 : COPY crypto /orderer/crypto
COPY failed: stat /var/lib/docker/tmp/docker-builder498873981/crypto: no such file or directory
Pulling insurance-ca (insurance-ca:)...
The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

It seems the docker-builderXXXXXXXXX/crypto files are not available.
